gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -428,7 +428,7 @@ gst_tensor_src_iio_stop (GstBaseSrc * src)
 static gboolean
 gst_tensor_src_iio_event (GstBaseSrc * src, GstEvent * event)
 {
-  //FIXME: fill this function
+  /* No events to be handled yet */
```

Comment:
Please use a **/**  .... */** style to generate doxygen document correctly ASAP.

Reply:
Yes, on it.